src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -180,6 +187,14 @@ def _describe(self, catalog: str, database: str, table: str) -> Table | None:
             logger.error(f"Couldn't fetch information for table {full_name} : {e}")
             return None
 
+    def unset_upgraded_to(self, database: str | None = None, name: str | None = None):
+        filter_exp = " "
```

Comment:
Bug: empty database and empty table would cause all records to be modified. Fix and add integration test to protect from it

Reply:
That's desirable.
If you want to revert the whole HMS.

Reply:
This requires a confirmation prompt from user and a comment in the code.

Reply:
OK, will do. Remember initially we were going to do the whole HMS. We added the database/table option as an afterthought.
Can I use prompts in CLI (like in installer)